.golangci.yml

```diff
@@ -28,3 +29,14 @@ linters-settings:
       - name: superfluous-else
       - name: modifies-parameter
       - name: unreachable-code
+
+  depguard:
+    list-type: blacklist
+    include-go-root: false
+    packages:
+      - log
+      - github.com/pingcap/errors
```

Comment:
Should this be `github.com/juju/errors`?
BTW, seems the lint check CI does not work.

Reply:
/cc @hi-rustin Do you have any idea about the check CI?

Reply:
It works locally.

```sh
make check  
check-copyright
gofmt (simplify)
tools/bin/gofumports -l -w $(find . -name '*.go' -type f | grep -vE 'vendor|kv_gen|proto|pb\.go|pb\.gw\.go') 2>&1 | awk '{ print  } END { if (NR > 0) { exit 1  }  }'
run shfmt
tools/bin/shfmt -d -w .
tools/bin/golangci-lint run --timeout 10m0s --skip-files kv_gen --skip-dirs dm
WARN [runner/nolint] Found unknown linters in //nolint directives: exported 
tests/integration_tests/bank/case.go:26:2: `github.com/pingcap/errors` is in the blacklist (depguard)
        "github.com/pingcap/errors"
        ^
tests/mq_protocol_tests/framework/docker_compose_op.go:22:2: `github.com/pingcap/errors` is in the blacklist (depguard)
        "github.com/pingcap/errors"
        ^
tests/mq_protocol_tests/framework/dsl.go:21:2: `github.com/pingcap/errors` is in the blacklist (depguard)
        "github.com/pingcap/errors"
        ^
make: *** [Makefile:210: check-static] Error 1
```